README.md

```diff
@@ -112,20 +112,49 @@ Please note that
  * ``-pl/-projects`` option specifies the list of projects that you want to install
  * ``-am/-also-make`` options specifies that dependencies should also be built
 
-## Include core components as subpackage into your own project maven build
+## Include core components into your own project maven build
 
-The released version of the core components are available on the public maven repository at https://repo.adobe.com. To include the
-core components package into your own project maven build you can add the dependency
+To add core components to your project, you will need to add it to your maven build.
+The released version of the core components are available on the public maven repository at https://repo.adobe.com.
```

Comment:
since version 2.2 we publish the released artifacts on maven central https://repo1.maven.org/maven2/com/adobe/cq/core.wcm.components.all/2.4.0/ and not on repo.adobe.com anymore.

Reply:
Updated, squashed and pushed!